mystyle.css

```diff
@@ -1,9 +1,92 @@
+p{
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+h1{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+    font-weight: 400;
+}
+
+input{
+    flex: auto;
+}
+
 .topnav{
     display: flex;
+    flex: auto;
+    justify-content: space-between;
 }
 .left-items{
-    
+    justify-content: flex-start;
+}
+
+.central-items {
+    display: flex;
+    justify-content: center;
+    flex:auto;
+}
+#search-input{
+    flex:auto;
+}
+.right-items {
+    justify-content: flex-end;
+}
+
+aside {
+    width:30%;
+    padding-left: 10px;
+    float: right;
+}
+#thumbox{
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
```

Comment:
flex-start is the default value of justify-content. it's ok to put the property  but you're making unnecessarily heavier your style file. (optional)
> in this simple project doesn't make the difference but imagine it in larger projects